test/Fail/QuoteHidden.err

```diff
@@ -1,4 +1,4 @@
-QuoteHidden.agda:6.5-16: error: [CannotQuote.Hidden]
+QuoteHidden.agda:6.5-18: error: [CannotQuote.Hidden]
```

Comment:
It feels these range changes are an improvement, because now the final braces of arguments passed as hidden/instance are included in the range.